### ISSUE TITLE
Lock screen does not work after logout

### ISSUE THREAD

**Issue by saidbakr** [elementary/wingpanel-indicator-session#99]
**Symptoms** 
Loss the ability to logout, shutdown and lock screen after performing Logout and relogin again.

**Procedures**

1. Start up the system.
2. Login to your account.
3. From the most right top corner press the shutdown icon in the system tray to get the menu.
4. From that menu choose Log Out.
5. Relogin again to get your desktop.
6. From the menu regarded in **3** try to Lock or to shutdown. The result is no shutdown occurred. The only way to shut down the computer is to use command `sudo shutdown now` from the terminal. I have to use `sudo` without `sudo` it does not work.

```
said@L850
         eeeeeeeeeeeeeeeee           OS: elementary OS 5.1 hera
      eeeeeeeeeeeeeeeeeeeeeee        Kernel: x86_64 Linux 5.0.0-37-generic
    eeeee  eeeeeeeeeeee   eeeee      Uptime: 1h 42m
  eeee   eeeee       eee     eeee    Packages: 2188
 eeee   eeee          eee     eeee   Shell: bash 4.4.20
eee    eee            eee       eee  Resolution: 2806x900
eee   eee            eee        eee  DE: GNOME 
ee    eee           eeee       eeee  WM: Mutter(Gala)
ee    eee         eeeee      eeeeee  WM Theme: elementary
ee    eee       eeeee      eeeee ee  GTK Theme: elementary [GTK2/3]
eee   eeee   eeeeee      eeeee  eee  Icon Theme: Papirus
eee    eeeeeeeeee     eeeeee    eee  Font: Open Sans 9
 eeeeeeeeeeeeeeeeeeeeeeee    eeeee   CPU: Intel Core i5-2450M @ 4x 3.1GHz [57.0°C]
  eeeeeeee eeeeeeeeeeee      eeee    GPU: AMD/ATI Thames [Radeon HD 7500M/7600M Series]
    eeeee                 eeeee      RAM: 1062MiB / 7933MiB
      eeeeeee         eeeeeee       
         eeeeeeeeeeeeeeeee          
```

**Comment by cassidyjames** [comment]
@RubenSibon that sounds like a completely separate issue, please file it as such. Thank you!

**Comment by agentwolf44** [comment]
I also have this exact same issue.
Additionally, I've also noticed that when I re-login it'll often hang on a black screen for some time and after that I won't be able to logout/shutdown. 

I'm not sure if this is part of the same issue, but after putting my laptop to sleep recently I tried to login and it wouldn't show the desktop (nor any other open programs) and then asked me to login again, after which still nothing showed (although the wallpaper did). This caused me to lose data on open programs after restarting (I know, I know, I should've saved. But it's still frustrating).
